frontend/src/views/cluster/ExposedService/ExposedServiceSideBar.vue

```diff
@@ -9,21 +9,43 @@ included in the LICENSE file.
     <template v-slot="{ open }">
       <disclosure-button as="div" class="disclosure">
         <div class="title">
-          <p class="title__name truncate">Exposed Services</p>
-          <t-icon icon="arrow-up" class="title__icon" :class="{'rotate-180': open}"/>
+          <p class="title-name truncate">Exposed Services</p>
+          <div class="expand-button">
+            <t-icon
+                class="w-6 h-6 hover:text-naturals-N13 transition-color transition-transform duration-250"
+                :class="{'rotate-180': !open}"
+                icon="drop-up"
+                />
+          </div>
         </div>
       </disclosure-button>
       <disclosure-panel>
         <template v-if="exposedServices.length > 0">
           <t-menu-item
-              v-for="service in exposedServices"
+              v-for="service in filteredExposedServices"
               :key="service.metadata.id"
               :route="service.spec.url"
-              :name="service.spec.label"
+              :name="service.spec.label!"
               :icon-svg-base64="service.spec.icon_base64"
               icon="exposed-service"
               regular-link
           />
+          <div class="flex gap-4 items-center pl-6 pr-5 text-xs" v-if="errors.length">
+            <t-icon icon="warning" class="ml-0.5 text-yellow-Y1"/>
+            <div class="text-yellow-Y1 flex-1 truncate">
+              {{ pluralize('service', errors.length, true) }} {{ errors.length === 1 ? 'has' : 'have' }} errors
```

Comment:
Is this JSX or some other vue feature? Sure looks like JSX to me :p

Reply:
If you mean computation, that's pretty old syntax that was there since Vue 2 at least.